buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -195,8 +195,8 @@
     "BuildConfiguration": {
       "value": "Release"
     },
-    "PB_BuildArguments": {
-      "value": "/p:TargetArchitecture=$(PB_TargetArchitecture) /p:ConfigurationGroup=$(BuildConfiguration) $(PB_AdditionalBuildArguments)"
+    "PB_CommonMSBuildArgs": {
```

Comment:
You should add `PB_PortableBuild=false` to the definition so that there is an obvious default value if someone tries to run the build definition, or it's not passed from pipeline.json